### ISSUE TITLE
How about add xy plot model block weight?

### ISSUE THREAD

**Issue by xueqing0622** [ltdrdata/ComfyUI-Inspire-Pack#26]
This is your good tutor to use your "xy plot lora block weight " to get perfect fixed  LBW
https://www.youtube.com/watch?v=X9v0xQrInn8&t=252s
https://github.com/ltdrdata/ComfyUI-extension-tutorials/blob/Main/ComfyUI-Inspire-Pack/tutorial/LoraBlockWeight.md
![image](https://github.com/ltdrdata/ComfyUI-Inspire-Pack/assets/114840118/29610e26-742e-4250-9d1e-0dd7cb053ea0)

How about add "xy plot model block weight"?
![image](https://github.com/ltdrdata/ComfyUI-Inspire-Pack/assets/114840118/d3642a9d-85cb-4394-96bd-e44be60bebbf)

**Comment by ltdrdata** [comment]
I'll consider adding it later.

**Comment by xueqing0622** [comment]
Thx!

**Comment by xueqing0622** [comment]
If you have the time, please move forward with this feature.
by the way, The author of efficiency-nodes-comfyui doesn't seem to be updating it anymore, so this is the one who does now.
https://githubfast.com/jags111/efficiency-nodes-comfyui

**Comment by ltdrdata** [comment]
> If you have the time, please move forward with this feature. by the way, The author of efficiency-nodes-comfyui doesn't seem to be updating it anymore, so this is the one who does now. https://github.com/jags111/efficiency-nodes-comfyui
> 
> or use ttn, it support xy plot too https://githubfast.com/TinyTerra/ComfyUI_tinyterraNodes/


I am contemplating the best approach for model merging, considering the substantial memory consumption involved.

**Comment by xueqing0622** [comment]
Indeed, when merging two sdxl models, my 12G card stutters for a long time, sd1.5 doesn't have this problem. 
I'm looking forward to the results of your research.
